gitlabform/lists/projects.py

```diff
@@ -112,6 +100,21 @@ def _get_projects(self, target: str, groups: Groups) -> Projects:
                 OmissionReason.ARCHIVED,
                 archived_projects_from_configuration_not_from_groups,
             )
+        else:
+            # in all other cases, we also need to look for projects in the config
+            # that are being transferred to a different namespace
+            for project in projects_from_configuration_not_from_groups:
+                project_transfer_source = self._get_project_transfer_source(project)
+
+                if project_transfer_source:
+                    if self._get_source_project(project, project_transfer_source):
+                        projects.add_requested([project])
+                    else:
+                        fatal(
+                            f"""Configuration contains project {project} to be transferred from {project_transfer_source}
+                                but the source project cannot be found in GitLab!""",
+                            exit_code=EXIT_INVALID_INPUT,
+                        )
```

Comment:
We have the same logic in `_get_single_project` function. But we're not setting similar log and exit code there. Should it be done in that function too? If not, could you clarify why?

Also, couldn't we just call that function for each projects in the loop?

Reply:
@long-wan-ep - did you get a chance to look into this?

Reply:
You're right, we can reuse it here and also the error handling here should be in that method, updated.

Reply:
Thanks for the update.